pyproject.toml

```diff
@@ -85,6 +85,8 @@ torch = ">=1.8.0"
 torchmetrics = ">=0.6.0"
 tqdm = ">=4.56.0"
 typing_extensions = {version = "*", python = "<3.8", optional = true}
+huggingface_hub = "*"
+dataclasses_json = "*"
```

Comment:
is this dependency strictly necessary? if it does something simple it would be better to implement that simple thing

Reply:
It was not-simple enough that I chose to avoid re-inventing the wheel, kind of the point of all these Python modules :)

Reply:
Dupe of another comment